src/components/SingleEvent/Admin/PartyForm.js

```diff
@@ -187,6 +197,22 @@ class PartyForm extends Component {
     })
   }
 
+  componentDidMount() {
+    getEtherPrice().then(r => {
+      if (r && r.result && r.result.ethusd) {
+        const unit = 10 // $10 as a guide price
+        const price = parseFloat(r.result.ethusd)
+        this.setState({ price: price })
+        if (!this.state.deposit) {
+          const ethCommitment = (unit / price).toFixed(2)
+          this.setState({ deposit: ethCommitment })
+        }
+      } else {
+        // falls back to default
+        this.setState({ deposit: 0.02 })
+      }
+    })
```

Comment:
We should catch the function call just in case the fetch throws and then in that throw we can also `setState` to fallback to 0.02

Reply:
done